assets/_scss/default/body-main-styles.scss

```diff
@@ -6,37 +6,34 @@
 :root {
   @include get_scheme(body-main-styles-colors-);
 }
-
 html {
   /* for comment page_bottom_Check */
   height: 100%;
   overflow-y: auto;
   /* font size base definition */
   font-size: 10px;
 }
-
 @media (max-width: 500px) {
   html {
     font-size: 9px;
   }
 }
-
 @media (max-width: 360px) {
   html {
     font-size: 8px;
   }
 }
-
 body {
-  color: var(--body-text-color);
+  color: black;
```

Comment:
It would be interesting to add the color using a variable. Like it was before this change.